### ISSUE TITLE
Flag emoji are broken on Chrome 131 on Windows 11... and ED (2025012301) on MacOS

### ISSUE THREAD

**Issue by ara4n** [element-hq/element-web#28500]
### Steps to reproduce

🏳️‍🌈🏳️‍⚧️🏴‍☠️ is rendered as:
![image](https://github.com/user-attachments/assets/9a924e26-de6a-4d5e-9174-73ef1b0a6c4a)

rather than:

![image](https://github.com/user-attachments/assets/857ab12b-28d7-4628-b714-bd9ac9b75242)

(as seen on macOS 15.1.1 and Chrome 131.0.6778.70)

see https://matrix.to/#/!IemiTbwVankHTFiEoh:matrix.org/$MKDDKlox-0HdM2XsqoFYD-M3CyWGpjItUKTDjW0jkaM?via=matrix.org&via=mozilla.org&via=envs.net

Circumstantially, this *may* have broken at the same time that Chrome fixed https://github.com/element-hq/element-web/issues/20043 (https://bugs.chromium.org/p/chromium/issues/detail?id=964527).

#### What did you expect?

Flags emoji to not get split apart into components.

#### What happened instead?

Whichever dialect of the twemoji font Windows uses seems to render wrong on Chrome 131 + Windows 11

### Operating system

Windows 11, version 23H2

### Browser information

chrome: Version 131.0.6778.86 (Official Build) (64-bit)

### URL for webapp

app.element.io

### Application version

v1.11.86

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by kilimnik** [comment]
Same Issue on Ubuntu 24.04 using Chrome Version 131.0.6778.108 (Official Build) (64-bit)

**Comment by ara4n** [comment]
This looks likely to be https://issues.chromium.org/issues/383824583. See also https://github.com/googlefonts/nanoemoji/issues/449

**Comment by ara4n** [comment]
Right, this can be worked around by defining additional ligatures in the COLR font without VARIATION SELECTOR-16 (U+FE0F) codepoints present.  For instance, to get the trans flag to work, you need to add:

`<Ligature components="u200d,u26a7" glyph="u1f3f3_fe0f_200d_26a7_fe0f"/>`

alongside the existing:

`<Ligature components="ufe0f,u200d,u26a7,ufe0f" glyph="u1f3f3_fe0f_200d_26a7_fe0f"/>`

**Comment by ara4n** [comment]
fixed by https://github.com/matrix-org/twemoji-colr/commit/94c684843e3d6739f3976a0030d2121329d5eb54 hopefully

**Comment by ara4n** [comment]
yay! as seen on develop Element version: 95da3834f288-js-fd9a44e70153 with Chrome 131.0.6778.265 on macOS 15.2 (24C101):

![Image](https://github.com/user-attachments/assets/ff870209-5d1b-4343-8a14-9012dbd697d4)
